src/bindings/jna/README.md

```diff
@@ -46,7 +48,7 @@ libelektra.jar), e.g.:
 export CLASSPATH=".:/usr/share/java/libelektra.jar:/usr/share/java/jna.jar"
 ```
 
-Then you can compile and run [HelloElektra](HelloElektra.java):
+Then you can compile and run [HelloElektra](hello/src/main/java/HelloElektra.java):
 
 ```sh
 javac HelloElektra.java && java HelloElektra
```

Comment:
Did this work for you?

Reply:
Yes, that works for me (provided I have set the `CLASSPATH` correctly as mentioned above).